### DISCUSSION TITLE
Encountered error while trying to install a package: pybullet

### DISCUSSION THREAD

**Discussion by Clarkszw** [fury-gl/fury#750] [Q&A] [answered]
I am following the steps to install all required packages for the documentation generation:

`pip install -U -r requirements/docs.txt`

And it gives me this error output:

```
running build_ext
      building 'pybullet' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> pybullet

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.
```
I am using Windows 11 for this installation.

**Comment by Clarkszw** [comment, marked as answer]
I have solved this issue by using the link in the output and download Microsoft C++ Build Tools:
https://visualstudio.microsoft.com/visual-cpp-build-tools/

Installed not only the default, but also 
**Select: Workloads → Desktop development with C++**

Although `pybullet` is installed successfully now, I have installed more than 7GB of Visual Studio related build tools. Does this sounds reasonable?

**Reply by skoudoro** [reply]
no, does not sound reasonable but can be useful later. you might ask pybullet to create a python wheel which will avoid you to build locally their package. Pybullet is mainly in C++ with a thin wrapper in python. I am not used to windows so I do not know the size of their compiler tools. 
Sometimes, it is needed to create an issue on pybullet and request a new package to avoid this.

**Reply by Clarkszw** [reply]
Hi @skoudoro, thanks for your reply:)
I am not quite understand about the meaning of "a python wheel". I will search around and learn:)
